ci/cloudbuild/builds/lib/features.sh

```diff
@@ -32,7 +32,7 @@ function features::always_build() {
     pubsub
```

Comment:
You seem to have changed the permissions for this file.  Was that intentional?  I think the previous permissions were fine, and arguably "safer"?

Reply:
d'oh, unintentional. Good catch. Fixed.